<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mini_usb_count_qt</class>
 <widget class="QMainWindow" name="mini_usb_count_qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mini USB counter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonUpdate_gate">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>70</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Gate time (ms)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to counter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controler Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSerialBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="control_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_freq">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>310</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>240</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gate_box">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>60</y>
      <width>136</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_gate">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>235</y>
      <width>506</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gate time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>235</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Counts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>300</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frequency </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>166</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Channel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="channel_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>125</y>
      <width>56</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonUpdate_channel">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>130</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>295</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Channel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_channel">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>300</y>
      <width>506</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboLogicBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>166</width>
      <height>33</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>NIM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TTL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>166</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logic type</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="plotWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>355</y>
      <width>986</width>
      <height>366</height>
     </rect>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
